scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -179,8 +179,9 @@ declare -gA geoIP=()
 [[ -z ${PEER_LIST_CNT} ]] && PEER_LIST_CNT=6
 
 # Style
-width=63
-second_col=$((width/2 + 3))
+width=67
+second_col=$((width/3 + 12))
+third_col=$((second_col + 16))
```

Comment:
This wasn't exactly what I had in mind. 
I want to be able to have both a two and three-column view.
So something like this 
* start position of first col not relevant
* +3 is used due to the fact that three chars go away at the start of the line
```
width=67
two_col_second=$((width/2 + 3))
three_col_second=$((width/3 + 3))
three_col_third=$(((width/3)*2 + 3))
```
with these you can build both a two-column view and a three-column view.